Task6.1/Author.cs

```diff
@@ -0,0 +1,37 @@
+﻿namespace BookCatalogTask
+{
+    public class Author
+    {
+        private string _firstName;
+        private string _lastName;
+
+        public string FirstName
+        {
+            get => _firstName;

```

Comment:
try to use const int MAX_LENGTH = 200
init => ValidateAndSetProperty(value, MAX_LENGTH, nameof(FirstName), ref _firstName);

Reply:
Fixed :)